frontend/.gitignore

```diff
@@ -11,4 +11,4 @@ node_modules/
 /blob-report/
 /playwright/.cache/
 visual-dashboard.spec.js-snapshots
-
```

Comment:
This has changed in the @belen-albeza last PR https://github.com/penpot/penpot/pull/4767/files#diff-8e548ede50532796f028e75d8f3384d5bdb57b9fbe2bda95521774d0c46d0f97R13 so probably it's not necesary